rollup.config.js

```diff
@@ -1,17 +1,38 @@
+const path = require('path');
 const resolve = require('rollup-plugin-node-resolve');
 const commonjs = require('rollup-plugin-commonjs');
 const babel = require('rollup-plugin-babel');
 const replace = require('rollup-plugin-replace');
-const { uglify } = require('rollup-plugin-uglify');
+const { terser } = require('rollup-plugin-terser');
 const json = require('rollup-plugin-json');
 const builtins = require('rollup-plugin-node-builtins');
 const globals = require('rollup-plugin-node-globals');
 const flow = require('rollup-plugin-flow');
 const filesize = require('rollup-plugin-filesize');
 const babelOptions = require('./babel.config');
+const pkg = require(path.join(process.env.INIT_CWD, 'package.json'));
 
 const env = process.env.NODE_ENV;
 const name = process.env.npm_package_name;
+const format = process.env.npm_lifecycle_event.split(':')[1];
```

Comment:
Sadly rollup does not expose its own CLI option again on the `process.env`. This is a kind of CoC approach to extract the format from the lifecycle.